src/ol/renderer/canvas/canvaslayerrenderer.js

```diff
@@ -192,19 +192,23 @@ ol.renderer.canvas.Layer.prototype.getImageTransform = goog.abstractMethod;
 
 /**
  * @param {olx.FrameState} frameState Frame state.
+ * @param {number=} opt_offsetX Offset on the x-axis in view coordinates.
  * @protected
  * @return {!goog.vec.Mat4.Number} Transform.
  */
-ol.renderer.canvas.Layer.prototype.getTransform = function(frameState) {
+ol.renderer.canvas.Layer.prototype.getTransform =
+    function(frameState, opt_offsetX) {
   var viewState = frameState.viewState;
   var pixelRatio = frameState.pixelRatio;
+  var offsetX = goog.isDef(opt_offsetX) ? opt_offsetX : 0;
```

Comment:
We could avoid the test by making the `offsetX` argument non-optional, and passing `0` when no offset is wanted.